app/models/find_api/provider.rb

```diff
@@ -0,0 +1,13 @@
+module FindAPI
+  class Provider < FindAPI::Resource
+    RECRUITMENT_CYCLE_YEAR = ENV.fetch('RECRUITMENT_CYCLE_YEAR') { 2020 }
```

Comment:
if you move the constant up to the `FindApi` module, we can remove the duplication
`RECRUITMENT_CYCLE_YEAR = ENV.fetch('RECRUITMENT_CYCLE_YEAR') { 2020 }`

Reply:
Sounds like a good idea, will try this